main.go

```diff
@@ -264,14 +251,19 @@ func processGitHubWebhook(ctx *gin.Context, payload []byte, githubClient *github
 
 		// First check if the PR has been merged. If so, stop
 		// the pipeline, and do nothing else.
-		if err = stopBuildsOfStalePRs(log, pr, conf); err != nil {
+		if err := stopBuildsOfStalePRs(log, pr, conf); err != nil {
 			log.Errorf("Failed to stop a stale build after the PR: %v was merged or closed. Error: %v", pr, err)
 		}
 
 		// Keep the OS and Enterprise repos in sync
-		if err = syncIfOSHasEnterpriseRepo(log, conf, pr); err != nil {
+		if err := syncIfOSHasEnterpriseRepo(log, conf, pr); err != nil {
 			log.Errorf("Failed to sync the OS and Enterprise repos: %s", err.Error())
 		}
+
+		// If the pr was merged, suggest cherry-picks
+		if err := suggestCherryPicks(log, pr, githubClient, conf); err != nil {
+			log.Errorf("Failed to suggest cherry picks for the pr %v. Error: %v", pr, err)
+		}
```

Comment:
This has to be done above line 240, or else won't be processed for PRs from contributors.

Reply:
right, done